public/resources/styles/main.css

```diff
@@ -0,0 +1,634 @@
+/* Font Import */
+@import url('https://fonts.googleapis.com/css?family=Fredoka+One|Monoton&display=swap');
+
+/* Background Image Import */
+html {
+    background: url(../images/desktop-bg.png) no-repeat center center fixed; 
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    -o-background-size: cover;
+    background-size: cover;
+    min-height: 100%;
+    z-index: 0;
+}
+
+/* Styling for non-mobile devices */
+body {
+    height: 100vh;
+    width: auto;
+    display: flex;
+    flex-direction: column;
+    margin: 0;
+}
+
+/* Styling for titles and headers */
+#title {
+    width: auto;
+    height: 125px;
+    background-color: black;
+    text-align: center;
+}
+
+h1 {
+    font-family: monoton;
+    color: white;
+    font-size: 4.1rem;
+    margin: auto;
+    padding-top: 1vh;
+}
+
+#home:hover {
+    cursor: pointer;
+    color: rgb(0, 105, 240);
+}
+
+h2 {
+    font-family: monoton;
+    color: white;
+    padding: 0;
+    margin: 0;
+}
+
+h3 {
+    font-family: monoton;
+    padding: 0;
+    margin: 0;
+}
+
+/* Styling for home page content */
+#main-container {
+    display: flex;
+    margin: 0 auto 0 auto;
+    max-height: 525px;
+}
+
+#restaurant-list {
```

Comment:
As I mentioned in my main comment, there is no ability to scroll down the list of restaurants on your sidebar. You could try adding an easy solution such as...
```css
div {
    overflow:scroll;
}